src/index.html

```diff
@@ -1,12 +1,408 @@
+    </header>
+
+    <main class="main">
+      <section class="card main__card">
+        <div class="card__container">
+          <article class="card__item" data-qa="card">
+            <img
+              class="card__foto"
+              src="images/imac.jpeg"
+              alt="foot product">
+            <h2 class="card__h2">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="card__cod">
+              Код товара: 195434
+            </p>
+            <div class="card__respond">
+              <div class="card__stars">
+                <div class="stars stars--4">
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                </div>
+              </div>
+              <p class="card__respond-item">Отзывов: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-p">
+                Цена:
+              </p>
+              <p class="card__price-item">
+                69 999 грн
+              </p>
+            </div>
+            <button class="card__button">
+              <a
+                class="card__link"
+                href="#"
+                data-qa="card-hover">
+                Купить
+              </a>
+            </button>
+          </article>
+        </div>
+
+        <div class="card__container">
+          <article class="card__item">
+            <img
+              class="card__foto"
+              src="images/imac.jpeg"
+              alt="foot product"
+            >
+            <h2 class="card__h2">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="card__cod">
+              Код товара: 195434
+            </p>
+            <div class="card__respond">
+              <div class="card__stars">
+                <div class="stars stars--4">
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                  <span class="stars__star"></span>
+                </div>
+              </div>
+              <p class="card__respond-item">Отзывов: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-p">
+                Цена:
+              </p>
+              <p class="card__price-item">
+                69 999 грн
+              </p>
+            </div>
+            <button class="card__button">
+              <a class="card__link" href="#">
+                Купить
+              </a>
+            </button>
+          </article>
+        </div>
+
+        <div class="card__container">
+          <article class="card__item">
```

Comment:
And do the same in all cards.
```suggestion
          <article class="card catalog__card">
```